buster/retriever/sqlite.py

```diff
@@ -44,3 +44,15 @@ def get_documents(self, source: str) -> pd.DataFrame:
         # Convert the results to a pandas DataFrame
         df = pd.DataFrame(rows, columns=[description[0] for description in results.description])
         return df
+
+    def get_source_display_name(self, source: str) -> str:
+        """Get the display name of a source."""
+        if source is "":
+            return "All"
```

Comment:
can we use a variable for "All" and share it with line 32 in `retriever/pickle.py`, e.g. 

ALL_SOURCES = "All Sources"

and import it from the other file?

Reply:
yes